docs/index.asciidoc

```diff
@@ -23,6 +23,44 @@ include::{include_path}/plugin_header.asciidoc[]
 The mutate filter allows you to perform general mutations on fields. You
 can rename, remove, replace, and modify fields in your events.
 
+[id="plugins-{type}s-{plugin}-proc_order"]
+===== Processing order
+
+By default, mutations in a config file are executed in this order:
```

Comment:
Is that just the default processing order for mutate actions that can be changed or is it a hard-coded processing order?

Reply:
I would read "By default" and assume that the processing order could be changed somehow.